LiverResections/qSlicerLiverResectionsModule.cxx

```diff
@@ -126,7 +126,7 @@ QStringList qSlicerLiverResectionsModule::categories() const
 //-----------------------------------------------------------------------------
 QStringList qSlicerLiverResectionsModule::dependencies() const
 {
-  return QStringList() << "LiverMarkups";
+  return QStringList() << "LiverMarkups" << "Markups";
```

Comment:
`LiverMarkups` already depends on `Markups` this looks redundant to me. Is there any reason to add it here explicitly @olevs, @gatangen ?

Reply:
The problem was that the python test failed without this addition. We couldn't get the dependencies loaded any other way

Reply:
I see. Let's keep it then.